enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/database/IdentityVerificationRepository.java

```diff
@@ -70,4 +70,23 @@ public interface IdentityVerificationRepository extends CrudRepository<IdentityV
             " AND id.status = com.wultra.app.enrollmentserver.model.enumeration.IdentityVerificationStatus.IN_PROGRESS " +
             " ORDER BY id.timestampLastUpdated ASC")
     Stream<IdentityVerificationEntity> streamAllInProgressClientEvaluations();
+
+    /**
+     * Return all identity verifications eligible for change to next state.
+     *
+     * @return identity verifications
+     */
+    @Query("SELECT id FROM IdentityVerificationEntity id WHERE" +
```

Comment:
Maybe we should consider a refactor of existing `DocumentsVerificationSyncTask` and `ClientEvaluationSyncTask`  tasks and cover all `status = IN_PROGRESS` here. But I see this would require adding new actions to the machine. So rather postpone.

Additionally some transitions from the `ACCEPTED` states could be maybe switched automatically by the state machine to the next state.

Reply:
I tried to limit `where` clause as much as possible. I may think about `ACCEPTED` once again.

Reply:
I recalled, we need to keep them because another jobs such as `DocumentsVerificationSyncTask`. We may consolidate it in an extra PR/issue.